src/Block/Renderer/ListItemRenderer.php

```diff
@@ -16,7 +16,9 @@
 
 use League\CommonMark\Block\Element\AbstractBlock;
 use League\CommonMark\Block\Element\ListItem;
+use League\CommonMark\Block\Element\Paragraph;
 use League\CommonMark\ElementRendererInterface;
+use League\CommonMark\Extension\TaskList\TaskListItemMarker;
```

Comment:
Kinda an unfortunate cyclic dependency here.

Reply:
Yeah, especially since the only purpose is to ensure the output exactly matches the GFM spec not adding a newline :upside_down_face:  Oh well